tiatoolbox/models/architecture/unet.py

```diff
@@ -168,11 +167,62 @@ def forward(self, x: torch.Tensor):
         return features
 
 
+def create_block(preact, kernels, input_ch, output_ch):
+    """Helper to create a block of Vanilla Convolution.
+
+    This is in pre-activation style.
+
+    Args:
+        preact (bool):
+            Requires preactivation layers.
```

Comment:
Whether to apply activation layer before the convolution layer. Should be True for ResNet blocks.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/345/commits/174ea015e1861c4c40877a3d25f70d30f3bed479